worker-document/src/main/java/com/hpe/caf/worker/document/scripting/specs/UrlScriptSpec.java

```diff
@@ -77,7 +79,7 @@ public boolean isStatic()
     @Override
     protected Reader openReader() throws IOException
     {
-        return new URLReader(url);
+        return new BufferedReader(new InputStreamReader(url.openStream()));
```

Comment:
You should always specify a character set when creating an `InputStreamReader`.  You never want to have any platform-specific behaviour.

```suggestion
        return new BufferedReader(new InputStreamReader(url.openStream(), StandardCharsets.UTF_8));
```